sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2374,14 +2374,14 @@ object functions {
    * @group datetime_funcs
    * @since 1.5.0
    */
-  def date_add(start: Column, days: Int): Column = withExpr { DateAdd(start.expr, Literal(days)) }
+  def date_add(start: Column, days: Int): Column = withExpr { AddDays(start.expr, Literal(days)) }
```

Comment:
I am willing to revert it back if there is no specific reason. This is about https://github.com/apache/spark/pull/14788#r77619318 - @hvanhovell